set_up_environment/first_steps.rst

```diff
@@ -0,0 +1,209 @@
+  The terminal is used for moving and copying files, starting software
+  and examining their error messages. To be precise, there is the terminal
+  emulator window and inside it another program, `shell`, which reads
+  user commands and acts according to them. If you are new to this, read the
+  `"Learning the shell" LinuxCommand.org <http://linuxcommand.org/lc3_learning_the_shell.php>`_
+  tutorial parts 1-6.
+
+Software prerequisites
+----------------------
+
+The first step to start authoring a course is to install some required software on your computer. The first two software
+you need to install are `git <https://git-scm.com/>`_ and `make <https://www.gnu.org/software/make/>`_. Sometimes, these
+two software are installed by default, but if **git** and **make** are not installed by default, you can install them by
+running the :ref:`commands below <install-git>` in your computer.
+
+.. code-block:: bash
+  :name: install-git
+  :caption: 1- Install Git
+
+  sudo apt update
+  sudo apt install git
+
+.. code-block:: bash
+  :caption: 2- Verify git version
+
+  git --version
+
+.. code-block:: bash
+  :caption: 3- Install build-essential, which includes make.
+
+  sudo apt update
+  sudo apt install build-essential
+
+.. code-block:: bash
+  :caption: 4- Verify make version
+
+  make --version
+
+You can also consult the official documentation for
+`installing git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_, and also for
+`installing make <https://pkgs.org/download/build-essential>`_.
+
+.. warning::
+  if you are using an Aalto's computer, you will need administrator rights to install the required software; otherwise,
+  you can ask an administrator to install them for you.
+  You can contact `Aalto IT services <https://www.aalto.fi/en/service-entities/it-services>`_ for requesting
+  administrator rights for the Aalto's computers.
+
+All other software runs inside `Docker <https://www.docker.com/>`_ containers. For short, Docker is the new software
+platform for running A+ and software related to it. This course will discuss Docker later.
+
+Install Docker for Mac/Windows or docker-ce & docker-compose for Linux.
+
+- `Docker Community edition <https://docs.docker.com/engine/installation/>`_
+- `Docker Compose <https://docs.docker.com/compose/install/>`_
+
+You also need to install a Text editor (`VS Code <https://code.visualstudio.com/>`_  is recommended). You can find more
+ information about the most convenient text editor to create *RST* documents
+ :ref:`at the end of this module <text-editor-and-ides>`.
+
+Cloning this codebase with git
+------------------------------
+
+A good way to start is to clone the `Aplus course Template`_ with
+git and begin to work on it. In Aalto, you could use
+`version.aalto.fi repo <https://version.aalto.fi/gitlab/course/aplus-manual>`_. Please note that if you don't have
+access, then you can request access to the *Aalto IT services*. In that case, you probably should be part of
```

Comment:
Yes, you are right. I will not mention https://version.aalto.fi here. GitHub is the right place to start. Regarding the wording, my idea here is to start using the [new course template](https://github.com/apluslms/aplus-course-template) so that we will not refer to the course manual any more. But first, we need to make the new course template official.